manifests/cf-manifest/cloud-config/021-cf-vm-extensions.yml

```diff
@@ -6,3 +6,9 @@ vm_extensions:
     cloud_properties:
       elbs:
         - (( grab terraform_outputs.cf_ssh_proxy_elb_name ))
+
+  - name: cf_rds_client
+    cloud_properties:
+      security_groups:
+        - (( grab terraform_outputs.default_security_group ))
```

Comment:
I don't think it's necessary to add the default security group here, it'll already be added to the base VM. (It wasn't necessary for the `ssh_proxy_elb` vm_extension above)